pywr/recorders/_recorders.pxd

```diff
@@ -1,5 +1,5 @@
 from pywr._component cimport Component
-from pywr._core cimport Timestep, AbstractNode, AbstractStorage, Storage, ScenarioIndex, Scenario
+from pywr._core cimport Timestep, AbstractNode, Node, AbstractStorage, Storage, ScenarioIndex, Scenario
```

Comment:
Is this needed?

Reply:
`cdef Node node = self._node`

It's needed for this type definition I think.